packages/builder/src/components/backend/DataTable/buttons/HideAutocolumnButton.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import { TextButton } from "@budibase/bbui"
+
+  export let hideAutocolumns = true
```

Comment:
Just noticed that this is defaulted to true here, and defaulted to false false in `Table`, and the real value is undefined in `DataTable`. Might be better to set the real variable to `true` in `DataTable` where it originates, and then leave both exports without an initial value since they're guaranteed to always be passed in the boolean value anyway. It probably doesn't cause any issues currently, but just looks potentially prone to a bug.